README.md

```diff
@@ -14,18 +16,62 @@ Algorithm output:
 - Objects cannot intersect;
 - Objects must fully fit into the area specified in the algorithm parameter;
 - There must be a distance of at least 10 pixels between objects;
-- The border of a polygon must be at least 4 pixels thick at its narrowest point;
-- The smaller of the two visible object dimensions must be at least 20 pixels;
+- The smaller of the two visible object dimensions must be at least 50 pixels;
 - The larger of the two visible object dimensions must be at least 100 pixels;
 - The angle of inclination of the chamber from the surface normal should be within 10 degrees;
-- The same object can be present in the photo several times. Objects will be considered independently in solving the placement problem;
-- The polygon must contain more than 2 corners (otherwise the algorithm will return false);
-- The large side of the image must be less than 4500 pixels;
-- The polygon must be drawn inside the sheet of paper so the border is no closer to the edge of the paper than 5 pixels;
-- The sheet of paper with a drawn polygon has to fit into the picture together with its borders.
+- The same object can be present in the photo several times. Objects will be considered independently in solving the 
+placement problem;
+- The polygon must contain more than 2 corners;
+- The large side of the image must be less than 4500 pixels.
 
 ## Items data and testing dataset
 
-Photos of possible items and background image (for second argument of algorithm): https://drive.google.com/drive/folders/1gk90RoNKPqP_lwWpY8cYFQpLThSHtY2m?usp=sharing 
+Photos of possible items and background image (for second argument of algorithm): 
+https://drive.google.com/drive/folders/1gk90RoNKPqP_lwWpY8cYFQpLThSHtY2m?usp=sharing 
 
-Dataset for testing: images with jpg extension and csv file of input/output data: https://drive.google.com/drive/folders/1lkSI2wThZ_rEAMk5LxqLqtVBevjNOp2e?usp=sharing 
+Dataset for testing: images with jpg extension and csv file of input/output data: 
+https://drive.google.com/drive/folders/1XGFnvI2nywZ0e6qcOO4kyAmzYbnEYxoU?usp=sharing 
+
+## Placer Algorithm
+The algorithm for finding objects:
+1. Get the contours of the image using the `skimage.filters.sobel`
+2. Remove the noise from the obtained image of the contour by binarization
```

Comment:
if this happens in def _get_objects_contours, then the order is incorrect, you have:
* gaussian
* grayscale
* sobel